app.js

```diff
@@ -9,11 +9,22 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function sum(a, b) { //eslint-disable-line
-
+  var a = 4;
+  var b = 7;
+  // console.log('Var a = ' + a + ' var b = ' + b)
+  var theSum = (a + b)
+  // console.log(theSum);
+  // console.log('The sum of ' + a + ' and ' + b + ' is ' + theSum + '!');
+  var sumString = ('The sum of ' + a + ' and ' + b + ' is ' + theSum + '.');
+  // console.log(sum);
+  return [theSum, sumString];
+  
 }
+// sum();
+// Yay - Passed with some tweaking but forgot to make a branch so add the branch and now uploading
 
 // Here is the test for sum(); uncomment it to run it
-// testSum(4, 7);
+testSum(4, 7);
 
```

Comment:
edited the app.js with the comment so I can practice teh push and pull into the master. YAY  completed